ee/fleetd-chrome/src/db.ts

```diff
@@ -42,7 +53,11 @@ export default class VirtualDatabase {
       new TablePrivacyPreferences(sqlite3, db)
     );
     VirtualDatabase.register(sqlite3, db, new TableScreenLock(sqlite3, db));
-    VirtualDatabase.register(sqlite3, db, new TableSystemInfo(sqlite3, db));
+    VirtualDatabase.register(
+      sqlite3,
+      db,
+      new TableSystemInfo(sqlite3, db, this.warnings)
```

Comment:
Why was only this table modified?
(Did I miss something about the other tables?

Reply:
This is the only table that is generated using multiple API calls. Single API call tables render error messages properly in the UI by just making the entire table/data fail. Since this table uses multiple API calls, the point is to allow successful data columns to be surfaced AND errors to be surfaced for any failing data columns.

Reply:
This solution is reusable if we end up building more tables that require multiple API calls to generate the data columns.

Reply:
👍

Reply:
At least `PrivacyPreferences` also uses multiple API calls, is that one covered?